src/s2/util/gtl/btree.h

```diff
@@ -815,7 +815,7 @@ class btree {
   };
 
   struct node_stats {
-    node_stats(ssize_t l, ssize_t i)
```

Comment:
Could you do `using size_type = typename Params::size_type;` and use that instead of repeating the `typename ...`?